hub/constraints/instruction-handling/call/finishing_touches/abort.lisp

```diff
@@ -28,13 +28,13 @@
                     (vanishes!    (shift    account/ROMLEX_FLAG      CALL_2nd_callee_account_row___abort_will_revert___row_offset))
                     (vanishes!    (shift    account/TRM_FLAG         CALL_2nd_callee_account_row___abort_will_revert___row_offset))
                     (vanishes!    (shift    account/RLPADDR_FLAG     CALL_2nd_callee_account_row___abort_will_revert___row_offset))
-                    (DOM-SUB-stamps---standard                       CALL_2nd_callee_account_row___abort_will_revert___row_offset
+                    (DOM-SUB-stamps---revert-with-current            CALL_2nd_callee_account_row___abort_will_revert___row_offset
```

Comment:
This was a real mistake. It's a revert `DOM/SUB` stamp we require

Reply:
indeed